lib/rails/generators/neo4j/install/templates/neo4j_rails5.rb

```diff
@@ -0,0 +1,9 @@
+# Configure NEO4J as the ORM
+Rails.application.config.generators do |g|
+  g.orm             :neo4j
+end
+
+# Configure where the neo4j database should exist
+Neo4j::Config.use do |config|
+  config[:storage_path] = "#{Rails.application.config.root}/db/neo4j-#{Rails.env}"
```

Comment:
Is `Rails.application.config.root` a `Pathname` object?  If so this could be:

    Rails.application.config.root.join("db/neo4j-#{Rails.env}")

But a couple of things about this: People could be using Neo4j in either server or embedded modes and if it's in server mode they could be setting it up in many ways (via the desktop app, downloading the `tar.gz`, docker, using the `rake` tasks, etc....).  So I'm not sure that it makes sense to set up a default `storage_path` or `url`